package.json

```diff
@@ -4,8 +4,8 @@
   "description": "crt portal",
   "main": "index.js",
   "scripts": {
-    "test:a11y": "pa11y-ci http://0.0.0.0:8000/report/",
-    "test:a11y-ci": "pa11y-ci http://127.0.0.1:8000/report/"
+    "test:a11y": "pa11y-ci --config .pa11y-local.json",
```

Comment:
@ilodidoj and I found that this command failed for us locally. We discovered the reason why is because my local environment does not include a `pa11y-screenshots` folder, and pa11y does not create the folder for automatically. Adding a `pa11y-screenshots/.gitkeep` folder will fix this.